src/flag_gems/runtime/backend/_nvidia/__init__.py

```diff
@@ -0,0 +1,33 @@
+import json
+import os
+
+from .ops import *  # noqa: F403
+
+
+class device:
```

Comment:
What's the relationship of this `device` and `device_ctx`.

If the only purpose of this class is to provide a static method, why bother?